auditlog/models.py

```diff
@@ -308,17 +308,20 @@ class Action:
         action. This may be useful in some cases when comparing actions because the ``__lt``, ``__lte``,
         ``__gt``, ``__gte`` lookup filters can be used in queries.
 
-        The valid actions are :py:attr:`Action.CREATE`, :py:attr:`Action.UPDATE` and :py:attr:`Action.DELETE`.
+        The valid actions are :py:attr:`Action.CREATE`, :py:attr:`Action.UPDATE`,
+        :py:attr:`Action.DELETE` and :py:attr:`Action.ACCESS`.
         """
 
         CREATE = 0
         UPDATE = 1
         DELETE = 2
+        ACCESS = 5
```

Comment:
Why 5? :confused:

Reply:
That was just a random choice to avoid conflicts in case other people also forked auditlog and fiddled with its actions. I am happy to change it to whatever you prefer.